CHANGELOG.md

```diff
@@ -10,13 +10,17 @@ possible (see our [Version Guarantees] for more info).
 
 These changes are available on the `master` branch, but have not yet been released.
 
-### Added
+## Added
```

Comment:
```suggestion
### Added
```

God damn it

Reply:
`##` Indicates the version.
`###` Indicates the type of change made.

Reply:
Added is not a version ;)